### ISSUE TITLE
write_dimensions encoding error

### ISSUE THREAD

**Issue by jamesrkg** [Quantipy/quantipy#929]
Emojis and probably other characters have no mapping to CP1252/Latin-1. This causes encoding errors in the Dimensions files. The following or similar needs to be done to prevent this.

```python
import codecs

#read input file
with codecs.open('file.csv', 'r', encoding = 'utf_8') as file:
    lines = file.read()  

#write output file
with codecs.open('file_ansi.csv', 'w', encoding = 'latin-1', errors='replace') as file:
    file.write(lines)
```

**Comment by AndersFreund** [comment]
New conclusion. 
This has nothing to do with the csv written from Quantpy. The problem is in the Dimension dms script.
The FileSystemObject cannot handle UTF-8. Instead I will use ADO Stream.

**Comment by jamesrkg** [comment]
Resolved by #934.
